pyo3-build-config/src/impl_.rs

```diff
@@ -1404,7 +1405,15 @@ fn cross_compile_from_sysconfigdata(
 ) -> Result<Option<InterpreterConfig>> {
     if let Some(path) = find_sysconfigdata(cross_compile_config)? {
         let data = parse_sysconfigdata(path)?;
-        let config = InterpreterConfig::from_sysconfigdata(&data)?;
+        // RFC: ignore lib_dir from sysconfig and use one from CrossCompileConfig input ?
+        let config = if cross_compile_config.lib_dir.is_some() {
+            InterpreterConfig {
+                lib_dir: cross_compile_config.lib_dir.as_ref().map(|path| path.display().to_string()),
+                ..InterpreterConfig::from_sysconfigdata(&data)?
+            }
+        } else {
+            InterpreterConfig::from_sysconfigdata(&data)?
+        };
```

Comment:
I think this is the correct place to put the change. I would do it like so:

```suggestion
        let mut config = InterpreterConfig::from_sysconfigdata(&data)?;
        if let Some(cross_lib_dir) = cross_compile_config.lib_dir_string() {
            config.lib_dir = Some(cross_lib_dir)
        }
```

Reply:
Thanks for you comment. Your suggestion is indeed cleaner than my draft, I applied it.